managers/graph-operations-manager.js

```diff
@@ -1,5 +1,18 @@
 class GraphOperationsManager {
-    constructor(props) {
+    constructor(config, services) {
```

Comment:
Shouldn't we also add config initialisation here?

Reply:
Valid point. I didn't do it since the config is not used anywhere in the code.
The reason why config is passed to the constructor is to keep consistency across all the Manager classes.

I guess we should change the order of `(config, services)` parameters across the library so that `config` would be optional, but it seems a bit unnecessary because of the priorities which we currently have.